tree/30_generic_methods/logger_rudder.cf

```diff
@@ -48,8 +48,10 @@ bundle agent logger_rudder(message, class_prefix)
         int        => getfields("^${current_technique_report_info.technique_name};;${c_class_prefix};;.*", "${expected_reports_file}", ";;", "report_data");
 
   classes:
+    (logger_rudder_final_resfile_repaired|logger_rudder_temp_resfile_kept.!logger_rudder_temp_resfile_repaired.final_resfile_exists)::
```

Comment:
ok, it does make sense, because od the final_resfile_exists being defined locally